### PR TITLE
Add development environment setup instructions

### PR DESCRIPTION
**Pull request by lf-**
I had a slight bit of difficulty figuring out the development environment, so let's document it!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,8 @@
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```
Comment (McTano) [inline README.md:1]: We can leave this line for now. Credit where credit's due. Just move it below the setup instructions.

File: README.md [hunk 2]
```diff
@@ -1,4 +1,8 @@
...
+## Development Environment Setup
+
+First, install the package manager `yarn` from your system package manager or with `npm install -g yarn`. Ensure you have git submodules set up for this project with `git submodule update --init`.
```
Comment (McTano) [inline README.md:3]: For the yarn installation instructions, we should just link to this page. The yarn team discourages using `npm` to install yarn, for *reasons*.
https://classic.yarnpkg.com/en/docs/install

Comment (McTano) [reply README.md:3]: I'd put the git submodules instruction on  a new line and maybe put the steps in bullet points.
Good job adding that. I always get confused when I freshly clone the repo, and have to look that command up.

Comment (orbit-stabilizer) [reply README.md:3]: `git clone --recurse-submodules ` also does the trick

Comment (lf-) [reply README.md:3]: Yep! I intentionally used a separate submodule command so it's the same to fix a clone you forgot to do recursively as it is to do it initially, but that works too.

Comment (lf-) [reply README.md]: Re: the yarn thing, I mistakenly thought we were using Yarn 2, for which that is the recommendation. I changed it.

**Comment by lf-** [comment]
I have implemented the suggested changes. We now have a more readable bulleted list of setup instructions.

The credit line has been reinstated but moved below a horizontal divider between the project specific stuff and the create-react-app documentation.

**Comment by McTano** [comment]
That is interesting. I didn't know there was a Yarn 2. We'll have to look
into upgrading.

**Comment by McTano** [comment]
Thanks a lot for improving the documentation!

**Pull request merged**